crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -144,7 +144,11 @@ impl SceneSpawner {
                     .ok_or_else(|| SceneSpawnError::NonExistentScene {
                         handle: scene_handle.clone_weak(),
                     })?;
-            scene.write_to_world(world, entity_map)
+            scene.write_to_world_with(
+                world,
+                entity_map,
+                &world.resource::<AppTypeRegistry>().clone(),
```

Comment:
We shouldn't need to clone here; we only need a reference.

Reply:
```
error[E0502]: cannot borrow `*world` as mutable because it is also borrowed as immutable
   --> crates/bevy_scene/src/scene_spawner.rs:147:13
    |
147 |             scene.write_to_world_with(world, entity_map, &world.resource::<AppTypeRegistry>())
    |             ^^^^^^-------------------^^^^^^^^^^^^^^^^^^^^^-----------------------------------^
    |             |     |                                       |
    |             |     |                                       immutable borrow occurs here
    |             |     immutable borrow later used by call
    |             mutable borrow occurs here
```

Reply:
yup